backend/django_core/static_src/components/minor/Comment/Error.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+    export let error: string;
+</script>
+
+<div class="flex w-full items-center justify-center md:gap-[1vw]">
+    <img
+        src="/static/public/images/gifs/sad.gif"
```

Comment:
You must be joking, use something akin to normal vite import

Reply:
> normal vite import

Vite import? what you mean?
this Looks good

Reply:
No, static import is messy, generally if you want to use something from django, you use static folder. Otherwise you directly import